.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,27 @@
+name: Run benchmarks
+
+on:
+  pull_request:
+
+jobs:
+  Benchmark:
+    runs-on: ubuntu-latest
+    env:
+      JULIA_DEBUG: BenchmarkCI
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.3
+      - uses: julia-actions/julia-buildpkg@master
```

Comment:
```suggestion
```

Maybe removing the build step would help.

It looks like this step alters `Project.toml` https://github.com/JuliaImages/ImageFiltering.jl/pull/148/checks?check_run_id=403841198#step:4:53

I think this is because `instantiate` updates `Project.toml` to the auto-generated version. So, this comment in `Project.toml` would be removed

https://github.com/JuliaImages/ImageFiltering.jl/blob/46c0445e239974791ebe6033ba77bf809d65a9fc/Project.toml#L31

Reply:
Okay lets try it, thanks!